Remotion/Web/Core/UI/Controls/ListMenuImplementation/Rendering/ListMenuRenderer.cs

```diff
@@ -74,10 +74,31 @@ public void Render (ListMenuRenderingContext renderingContext)
       RegisterMenuItems(renderingContext);
 
       renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Id, renderingContext.Control.ClientID);
-      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, CssClassListMenu);
-      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute2.Role, HtmlRoleAttributeValue.Menu);
+      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, "listMenuRoot");
+      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute2.Role, "region");
+
       if (IsDiagnosticMetadataRenderingEnabled)
         AddDiagnosticMetadataAttributes(renderingContext);
+
+      renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Div);
+
+      var renderHeading = renderingContext.Control.Heading is { IsEmpty: false };
+      if (renderHeading)
+      {
+        var heading = renderingContext.Control.Heading!.Value;
+        var headingID = renderingContext.Control.ClientID + "_Heading";
+        var tag = GetTagFromHeadingLevel(renderingContext.Control.HeadingLevel);
+        renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Id, headingID);
+        renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, "screenReaderText");
+        renderingContext.Writer.RenderBeginTag(tag);
+        heading.WriteTo(renderingContext.Writer);
+        renderingContext.Writer.RenderEndTag();
+
+        renderingContext.Writer.AddAttribute("aria-labelledby", headingID);
```

Comment:
please move this down to the other attributes for table. keeping it in a separate block means that the attribute will invariably get applied to the wrong element during the next change in the renderer (read: a bug waiting to happen)

From the item order, please move it to the other aria attributes.

Lastly, there's an `HtmlTextWriterAttribute2.Labelledby` attribute name constant already available.